grid-cli/internal/config/config.go

```diff
@@ -56,6 +56,10 @@ func (c *Config) Load(path string) error {
 // GetConfigPath returns the path of tf-grid configuration file
 func GetConfigPath() (string, error) {
 	configDir, err := os.UserConfigDir()
+	if _, err := os.Stat(configDir); os.IsNotExist(err) {
+		return "", errors.Wrap(err, "config directory does not exist")
```

Comment:
why not creating the dir if not exist?

Reply:
my point is, not having that directory created is by itself fishy, and that's probably the case when the user runs it as user "root", But I understand your reasoning